src/controllers/tripsController.js

```diff
@@ -40,13 +41,13 @@ class tripsController {
           returnDate
         },
         user: {
-          email, userID, passportNumber, gender, managerEmail, preferedLang
+          email, userID, passportNumber, gender, managerEmail, preferedLang, firstName,
```

Comment:
```suggestion
          email, id, passportNumber, gender, managerEmail, preferedLang, firstName,
```
@RusimbiP after rebasing change the userID to id